WalletWasabi.Gui/Controls/WalletExplorer/SendControlViewModel.cs

```diff
@@ -377,8 +382,8 @@ protected SendControlViewModel(Wallet wallet, string title)
 					IsBusy = false;
 				}
 			},
-			this.WhenAny(x => x.IsMax, x => x.AmountText, x => x.Address, x => x.IsBusy,
-				(isMax, amount, address, busy) => (isMax.Value || !string.IsNullOrWhiteSpace(amount.Value)) && !string.IsNullOrWhiteSpace(Address) && !IsBusy)
+			this.WhenAny(x => x.IsMax, x => x.AmountText, x => x.Address, x => x.IsBusy, x => x.HasErrors,
+				(isMax, amount, address, busy, hasErrors) => (isMax.Value || !string.IsNullOrWhiteSpace(amount.Value)) && !string.IsNullOrWhiteSpace(Address) && !IsBusy && !HasErrors)
```

Comment:
HasErrors was intentionally left out from here. The user can press the button and get the error. Without able to press it she has to figure out what is the problem. For example, the amount was not entered.

Reply:
removed.